### ISSUE TITLE
New bookmark is only based on domain name and ignores path

### ISSUE THREAD

**Issue by realgooseman** [raindropio/app#147]
**Describe the bug**
When I try to add a bookmark on a website the pop-up window doesn't always fetch the link path for page title/description/favicon. And when I'm trying to save another bookmark from that same website but with different path, it edits the first saved bookmark's link of that website instead of adding a new one.

**To Reproduce**
1. Go on a website.
2. Add bookmark of current website link (shortcut or save button).
3. Pop-up clipper occasionally shows base domain information with link (even if path has different page title/description/...).
4. Adding a new link for that same website but with different path, never creates new bookmark but edits link of first saved one.

**Expected behavior**
When adding a bookmark, the clipper should fetch all the information from that specific website's path. It should also create a new bookmark when saving a new link with different path from that same website instead of editing the existing bookmark.

**My system**
Windows 11 Professional : 26100.2161 24H2
Browser : Brave v1.71.118 (Chromium: 130.0.6723.70)
Chrome extension : Raindrop v5.6.53

**Additional context**
The problem only happens with Raindrop Chrome Extension, manually adding any link on Raindrop's web app via top right "Add" button works normally.

**Comment by realgooseman** [comment]
Hi @exentrich,

For example, Raindrop doesn't bookmark tracking URL's for this Belgian post parcel tracking website below.

**Main page URL :** https://track.bpost.cloud/btr/web/#/home?lang=en

**Tracking URL (fake link for privacy reasons) :** https://track.bpost.cloud/btr/web/#/home?lang=en&itemCode=329998745632147852365873&postalCode=1000

When trying to bookmark the second link (Tracking URL), Raindrop doesn't save that link but rather tries to edit the **_Main page URL_** if it's already bookmarked. If the Main page URL is not already bookmarked, Raindrop extension bookmarks the full **_Tracking URL_** normally but prevents to bookmark a different Tracking URL afterwards, it edits the previous one I bookmarked each time.

So basically, I can't save two different bookmark URL's from this same domain with Raindrop Chrome extension.

Didn't find other websites with this problem yet but maybe if they have a similar URL structure they might have that problem too. Can't point my finger at where the problem really comes from.

**Important note :** problem only happens with Raindrop Chrome Extension, manually adding any link on Raindrop's web app via top right Add button works normally.

**Comment by exentrich** [comment]
The issue is fixed now, sorry for inconvenience. Now such URL detected properly as unique URL
